jba/README.md

```diff
@@ -138,18 +189,21 @@ docker run hyperstyle-analysis-prod:<VERSION> poetry run tasktracker_task_filter
    ```
 
    **Required arguments**:
-   
-   - `preprocessed_course_data_path` — Path to .csv file with preprocessed data by [data_processing.py](src/processing/data_processing.py).
-   - `course_structure_path` — Path to .csv file with the course structure gathered by [prepare_course_data.py](src/processing/prepare_course_data.py).
-   
+
+    - `preprocessed_course_data_path` — Path to .csv file with preprocessed data
+      by [data_processing.py](src/processing/data_processing.py).
+    - `course_structure_path` — Path to .csv file with the course structure gathered
+      by [prepare_course_data.py](src/processing/prepare_course_data.py).
+
    Optional arguments:
-   
+
    | Argument                            | Description                                 |
-   |-------------------------------------|---------------------------------------------|
+         |-------------------------------------|---------------------------------------------|
```

Comment:
It seems there is a bug in the IntelliJ markdown formatter. Please undo the changes in this line